nixos/modules/misc/ids.nix

```diff
@@ -339,6 +339,7 @@
       rss2email = 312;
       cockroachdb = 313;
       zoneminder = 314;
+      unit = 315;
```

Comment:
No need to allocate static uid/gid's, you can just omit it and it will work too.